helm-charts/splunk-otel-collector/templates/config/_otel-k8s-cluster-receiver-config.tpl

```diff
@@ -82,6 +82,8 @@ processors:
   {{- end }}
 
   batch:
+    metadata_keys:
+      - X-SF-Token
```

Comment:
This is the only configuration where we don't need it. The cluster receiver doesn't get any push-based receivers.